### ISSUE TITLE
Specific version images and nightly build image

### ISSUE THREAD

**Issue by onukura** [gitbucket/gitbucket-docker#22]
It would be very useful if there are more specific images and ideally nightly build image like bellow.

```
gitbucket/gitbucket:latest
gitbucket/gitbucket:nightly
gitbucket/gitbucket:4.34
gitbucket/gitbucket:4.33
gitbucket/gitbucket:4.32
gitbucket/gitbucket:4.31
...
```

**Comment by takezoe** [comment]
Hmm. Tags are not applied to DockerHub? 🤔 

- https://github.com/gitbucket/gitbucket-docker/tags
- https://registry.hub.docker.com/r/gitbucket/gitbucket/tags

**Comment by onukura** [comment]
I can see only `latest`, `4`, `3`, `4.29.0` ...
For example latest ver. `4.34.0`, there is [tag](https://github.com/gitbucket/gitbucket-docker/releases/tag/4.34.0) but not in DockerHub.

**Comment by takezoe** [comment]
If I remember correctly, when I created 4.29.0 tag on GitHub, it was built on DockerHub automatically. I wonder why it doesn't work now. Maybe additional steps are necessary?

**Comment by onukura** [comment]
It should be like that normally when tag is created if a setting for build rules in Build configurations is enable.
![image](https://user-images.githubusercontent.com/26293997/89174549-fe977f80-d5c0-11ea-80d6-21d1ff4e857e.png)

**Comment by onukura** [comment]
@takezoe Or GitHub Actions might be an option for that.
I made a trial which use github actions for building tag and nightly build. What do you think about it?  
https://github.com/onukura/gitbucket-docker/tree/dev
(I removed folder structure because there is already [ver. 3 image](https://hub.docker.com/layers/gitbucket/gitbucket/3/images/sha256-f7337e13ba1537dd2f8164760892d5a8355a3db4f579fc48575c42743f106a60?context=explore) on Docker Hub)

**Comment by onukura** [comment]
@takezoe I made draft PR #23, please check when you have time.

**Comment by takezoe** [comment]
Tag build is not configured on DockerHub and I can't update the configuration for some reason. Maybe there is a problem in GitHub integration?

<img width="1099" alt="Screen Shot 2020-08-09 at 17 51 34" src="https://user-images.githubusercontent.com/1094760/89728560-0778cd00-da69-11ea-94eb-d724fa2872eb.png">

**Comment by onukura** [comment]
that is very strange... you can't delete them neither?

**Comment by takezoe** [comment]
I haven't tried that because I'm not sure if I can re-add them once I delete...

**Comment by onukura** [comment]
I see... So more because of this integration issue, why not moving to GitHub Actions(GAs) to publish images #23 ?? While docker hub integration build is passive, GAs can actively control release. And nightly build is easier than docker-hub integration. To do nightly build by docker-hub integration, you might need complicated Dockerfile.
please feel free to tell me if it's out of your scope. I will do this in my own repo! ;)

**Comment by takezoe** [comment]
There is no special reason to avoid using GitHub Actions. I just didn't have enough  time to take care of this recently. 🙇

- Disable automatd build on DockerHub
- Use GitHub Actions #23

shoud be fine!

**Comment by onukura** [comment]
@takezoe Great! I updated #23. Now ready for review. 
This GitHub actions has 3 triggers.

| dockerfile | trigger | docker image tag |
| --- | --- | --- |
| Dockerfile | push to master branch | `latest` |
| Dockerfile | tag | e.g. `4.34.0` |
| Dockerfile-nightly | scheduled (0 0 * * *) | `nightly` |

**Comment by takezoe** [comment]
Thanks! I will take a look.

**Comment by takezoe** [comment]
Implemented in #23.
